File: backend/src/main/java/br/com/ifsp/classify/dtos/update/ClassSessionUpdateDTO.java
package br.com.ifsp.classify.dtos.update;

import java.time.LocalDateTime;

public record ClassSessionUpdateDTO(
    String subjectTeacherId,
    String classroomUuid,
    LocalDateTime startTime,
    LocalDateTime endTime,
    ReportUpdateDTO report,
    String classUuid,
    String studentUuid
) {}

File: frontend/src/shared/dtos/class-session/ClassSessionUpdateDTO.ts
import type { ReportCreateDTO } from "../report/ReportCreateDTO";

export interface ClassSessionUpdateDTO {
    subjectTeacherId?: string;
    classroomUuid?: string;
    startTime?: string;
    endTime?: string;
    report?: ReportCreateDTO;
    classUuid?: string;
    studentUuid?: string;
}
